src/components/views/avatars/MemberPresenceAvatar.js

```diff
@@ -25,6 +25,7 @@ import dispatcher from "../../../dispatcher";
 import * as ContextualMenu from "../../structures/ContextualMenu";
 import SettingsStore from "../../../settings/SettingsStore";
 
+// This is just an avatar with a presence indicator for the user on it.
```

Comment:
... and the option to change the presence?

Reply:
@richvdh updated